packages/swapper/src/swappers/LifiSwapper/utils/transformLifiFeeData/transformLifiFeeData.ts

```diff
@@ -38,6 +39,10 @@ export const transformLifiStepFeeData = ({
       }
     }
 
+    if (name === LIFI_SHARED_FEES_STEP_NAME) {
+      return acc
+    }
+
```

Comment:
@NeOMakinG non-blocking but would be noice to comment what this is and why we have this check

Reply:
e3f53d4233